Cask

```diff
@@ -5,4 +5,6 @@
 
 (development
  (depends-on "noflet")
- (depends-on "helm"))
+ (depends-on "ag")
+ (depends-on "helm")
+ (depends-on "helm-ag"))
```

Comment:
Hmm. I totally missed this - if this is really the case, then someone made a mistake. I don't use `helm-projectile` at all, so I don't pay much attention the things that are submitted to it.